addons/stock/static/src/legacy_web_report/report.xml

```diff
@@ -1,9 +1,9 @@
 <template>
 
     <!-- Main layout (iframe) -->
-    <div t-name="report.client_action">
+    <t t-name="report.client_action">
```

Comment:
why changing this?

Reply:
It's not the same, here you remove a `<div>`, it could break stuff (the style for instance).